Competition2017/src/competition/subsystems/shooter_belt/ShooterBeltSubsystem.java

```diff
@@ -33,8 +35,10 @@ public ShooterBeltSubsystem(
         this.side = side;
         
         intakePowerProperty = propManager.createPersistentProperty("ShooterBelt intake power", 0.5);
+        intakeTracerPowerProperty = propManager.createPersistentProperty("ShooterBelt intake power", 0.25);
```

Comment:
this property has the same name as the one above it so there will only be one on the dashboard